src/test/java/org/broadinstitute/hellbender/tools/copynumber/CreateBinningIntervalsIntegrationTest.java

```diff
@@ -0,0 +1,221 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.engine.ReferenceDataSource;
+import org.broadinstitute.hellbender.utils.GenomeLocParser;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import htsjdk.samtools.util.IntervalList;
+import htsjdk.samtools.util.Interval;
+
+import java.io.File;
+import java.io.*;
+import java.io.IOException;
+
+
+public class CreateBinningIntervalsIntegrationTest extends CommandLineProgramTest {
+
+    private static final File REFERENCE = new File(b37_reference_20_21);
+    private static final GenomeLocParser GLP = new GenomeLocParser(ReferenceDataSource.of(REFERENCE).getSequenceDictionary());
+
+    // Test for separate intervals
+    @Test
+    public void simpleIntervalTest1() {
+        final int widthOfBins = 10000;
+        final int padding = 500;
+        final File outputFile = createTempFile("GATK-create-binning-intervals-test", ".tmp");
+        final String[] args = {
+                "-L", "20:3000-20000",
+                "-L", "20:200-1900",
+                "-R", REFERENCE.getAbsolutePath(),
+                "-" + CreateBinningIntervals.WIDTH_OF_BINS_SHORT_NAME, Integer.toString(widthOfBins),
+                "-" + CreateBinningIntervals.PADDING_SHORT_NAME, Integer.toString(padding),
+                "-O", outputFile.getAbsolutePath()
+        };
+        runCommandLine(args);
+
+        // read the contents of the output file and print them out
+        // printOutputFile(outputFile);
+
+        // read the bins from the output file
+        IntervalList binsFromFile = IntervalList.fromFile(outputFile);
+
+        // we expect the following result
+        IntervalList binsExpected = new IntervalList(binsFromFile.getHeader().getSequenceDictionary());
+        String contig = binsFromFile.iterator().next().getContig();
```

Comment:
This could be made final.  However, you should instead hard code the expected contig (`binsExpected.add(new Interval("20", 1, 2400));`, etc.).  With the test as written, your code could write the wrong contig and still pass, since it reads that contig back in and puts it in `binsExpected`.

Reply:
You're right, I corrected it.